.github/workflows/gradle.yaml

```diff
@@ -0,0 +1,33 @@
+name: Java CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - feature/*
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ inputs.platform }} @ ${{ github.event.pull_request.number || github.sha }}'
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
```

Comment:
CoAT uses Java 8 to compile, reference https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:build/android/gradle/java.jinja;l=28?q=p:cobalt%20JavaVersion.VERSION_1_8

It is backwards compatible, as long as the version here is greater than 8, it should work.

Reply:
This is reverted to version 11, because of the error
> Failed to apply plugin 'com.android.internal.application'.
   > Android Gradle plugin requires Java 11 to run. You are currently using Java 1.8.